backend/tests/unit/models/conftest.py

```
@@ -0,0 +1,244 @@
+        model_json = model_instance.model_dump_json()
+        
+        # Deserialize from JSON
+        reconstructed_model = model_class.model_validate_json(model_json)
+        
+        # Verify they're equal
+        assert model_instance == reconstructed_model
+        
+        return model_instance, model_json
+    
+    @staticmethod
+    def test_optional_fields(model_class, required_fields, optional_fields, valid_data):
```

Ruff (0.12.2)

Unused static method argument: `optional_fields`

(ARG004)